build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -230,9 +236,8 @@ steps:
   - task: PublishTestResults@2
     displayName: 'Publish Test Results test-results.xml'
     inputs:
-      testResultsFiles: 'test-results.xml'
-      searchFolder: '$(Build.SourcesDirectory)'
-    continueOnError: true
+      testResultsFiles: '*.xml'
+      searchFolder: '$(Build.ArtifactStagingDirectory)/test-results'
     condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
 
   - task: PublishCodeCoverageResults@1
```

Comment:
Do we really want to not continue on error for this and the publish test results? It seems like publishing code coverage and test results shouldn't necessarily block the build if they fail.

Reply:
the publish test results actually fails the step if any tests failed, so it make sense to block the build for it.